packages/wonder-blocks-accordion/src/components/accordion-section.tsx

```diff
@@ -158,7 +164,9 @@ type Props = AriaProps & {
  */
 const AccordionSection = React.forwardRef(function AccordionSection(
     props: Props,
-    ref: React.ForwardedRef<HTMLDivElement>,
+    // Using a button ref here beacuse the ref is pointing to the
+    // section header, which is a button.
+    ref: React.ForwardedRef<HTMLButtonElement>,
```

Comment:
I guess I could have done this a different way so that the main ref is to the container and there's a separate ref for the header button? I personally liked this approach better because I think it makes sense for people to want a ref to the button itself?

I tried seeing if `useImperativeHandle` would be appropriate here, but the ref types weren't matching up.

Reply:
I think this change makes sense to me, specially from the consumer perspective.